src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -258,7 +258,7 @@ fn source_render_operator<G, C>(
     scope: &mut G,
     config: RawSourceCreationConfig,
     source_connection: C,
-    resume_uppers: impl futures::Stream<Item = Antichain<C::Time>> + 'static,
+    resume_uppers: impl futures::Stream<Item = Antichain<C::Time>> + Send + 'static,
```

Comment:
What needed this?

Reply:
oh this was from something previous, will remove